src/test/java/BaseTest.java

```diff
@@ -38,6 +39,18 @@ static void setupClass() {
         WebDriverManager.chromedriver().setup();
     }
 
+    public WebDriver driver = null;
+    public String url = null;
+    public WebDriverWait wait = null;
+
+    public FluentWait fluentWait = null;
+    public Actions actions = null;
+    public ThreadLocal<WebDriver> threadLocal = null;
+
```

Comment:
**Please, remove code redundancies**

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

![image](https://user-images.githubusercontent.com/104761482/215411705-b4a0fa7d-d895-4ef3-ab9c-a82f0f4bdff2.png)